### PR TITLE
update access policy func tests

### PR DESCRIPTION
**Pull request by honourfish**

### CODE REVIEW

File: functests/execaccess_policies.py [hunk 1]
```diff
@@ -55,6 +58,52 @@
...
+
+        if subject["confirmation_status"] == "CONFIRMED":
+            return
+
+        sleep(POLL_INTERVAL)
+
+    raise AssertionError(f"subject: {subject['identity']} is not confirmed.")
+
+
+def reciporcal_subects(arch_1, arch_2):
```
Comment (eccles) [inline functests/execaccess_policies.py:79]: typo ...

File: functests/execaccess_policies.py [hunk 2]
```diff
@@ -55,6 +58,52 @@
...
+
+    subject_2 = arch_2.subjects.create(
+        "org1_subject",
+        self_subject_1["wallet_pub_key"],
+        self_subject_1["tessera_pub_key"],
+    )
+
+    # check the subjects are confirmed
+    poll_subject_confirmed(arch_1, subject_1["identity"])
+    poll_subject_confirmed(arch_2, subject_2["identity"])
```
Comment (eccles) [inline functests/execaccess_policies.py:102]: This is OK but better would be to add the confirm=   keyword to the subjects.create method...

I will approve this but would you like to try that?

**Review by eccles — CHANGES_REQUESTED** [review]

File: functests/execaccess_policies.py (hunk 1 above)
Comment (honourfish) [reply functests/execaccess_policies.py:79]: good spot! will address :)

File: functests/execaccess_policies.py (hunk 2 above)
Comment (honourfish) [reply functests/execaccess_policies.py:102]: yeh i will take a look :)

File: functests/execaccess_policies.py (hunk 1 above)
Comment (honourfish) [reply functests/execaccess_policies.py]: fixed!

File: functests/execaccess_policies.py (hunk 2 above)
Comment (honourfish) [reply functests/execaccess_policies.py:102]: due to the confirmation of subject creation needing an external call to be made using a different tenant, we will keep this as is for now, and raise a ticket for confirming subjects.

https://github.com/jitsuin-inc/archivist-python/issues/101

Comment (eccles) [reply functests/execaccess_policies.py:102]: Raise an issue on github

File: functests/execaccess_policies.py [hunk 3]
```diff
@@ -55,6 +58,52 @@
...
+
+        if subject["confirmation_status"] == "CONFIRMED":
+            return
+
+        sleep(POLL_INTERVAL)
+
+    raise AssertionError(f"subject: {subject['identity']} is not confirmed.")
+
+
+def reciprocal_subects(arch_1, arch_2):
```
Comment (eccles) [inline functests/execaccess_policies.py:79]: still a typo

**Review by eccles — CHANGES_REQUESTED** [review]

**Review by eccles — APPROVED** [review]

**Pull request merged**